src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -65,9 +70,12 @@ def plot_persistence_barcode(
     fontsize=16,
 ):
     """This function plots the persistence bar code from persistence values list
+    , a np.array of shape (N x 2) (representing a diagram 
+    in a single homology dimension), 
     or from a :doc:`persistence file <fileformats>`.
 
-    :param persistence: Persistence intervals values list grouped by dimension.
+    :param persistence: Persistence intervals values list grouped by dimension, 
+                        or np.array of shape (N x 2).
     :type persistence: list of tuples(dimension, tuple(birth, death)).
```

Comment:
I am proposing this documentation (to modified for the 2 other functions too), to explain that it can be an numpy.ndarray (N x 2), or a classical array:
```suggestion
    :param persistence: Persistence intervals values list. Can be grouped by dimension or not.
    :type persistence: an array of (dimension, array of (birth, death)) or an array of (birth, death).
```

Reply:
incorporated in last commit